src/main/java/siri/Deadline.java

```diff
@@ -0,0 +1,80 @@
+package siri;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+class Deadline extends Task {
+    private String dateTime;
+
+
+    private static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd-LLL-yyyy");
+    private LocalDate dlDate;
+    private LocalTime dlTime;
```

Comment:
Perhaps it's better to use full words for the variable names (e.g. `dueLocalDate` and `dueLocalTime`)?

Similar for `dlLst`, `eLst` etc